nhs/data/handling.py

```diff
@@ -106,39 +108,56 @@ def read_spreadsheets(
 
 
 @log_entry_exit(level="INFO")
-def standardize_names(df_dict: dict[str, pl.LazyFrame | None], census_metadata: pl.LazyFrame | None,
-                      census_code_col: str, abbreviation_column_name: str, long_column_name: str) -> dict[str, pl.LazyFrame | None]:
+def standardize_names(
+    df_dict: dict[str, pl.LazyFrame],
+    census_metadata: pl.LazyFrame,
+    census_code_col: str,
+    abbreviation_column_name: str,
+    long_column_name: str,
+) -> dict[str, pl.LazyFrame | None]:
     """
     Standardise the column names of a polar Lazy frame dictionary to make them more readable
 
-    This function will
-        1. Expand abbreviated names.
-        2. Converting to lower case.
-
-    If any parameters are None, this function returns prev_csv_pl_df without change. Keys in df_dict must have format 'G{int + letter}'.
-
+    This function will expand abbreviated names using a census metadata frame containing
+    mappings of abbreviated names to full names in the columns `abbreviation_column_name` and
+    `long_column_name`. The names are also converted to snake_case.
 
     Parameters:
     --------
     df_dict : dict[str, pl.LazyFrame]
-        A dictionary containing census data as value and filename as key or None. a dictionary of name-dataframe pair. Names must exist in the census_metadata[census_code_col]. All column names in the dataframes in df_dict with keys in format 'G{int + letter}'.
-    census_metadata : [pl.LazyFrame | None]
-        Census_metadata: Census metadata lazy frame with one standard sheet has a column called identification containing list of keys in the df_dict dictionary to standarise, and columns called abbreviated.... and long_column)names containing the abbreviated and long column names for data frames in the df_dict or None. If don't have abbreviated column, long column and census_code_col then df_dict won't be changed.
+        A dictionary containing census data as value and filename as key. Names
```

Comment:
You repeated yourself twice with `"A dictionary containing census data as value and filename as key or None."` and `"a dictionary of name-dataframe pair"`, try to keep the documentation concise. Also, `"All column names in the dataframes in df_dict with keys in format 'G{int + letter}'."` is NOT required.